internal/pkg/service/common/servicectx/servicectx_test.go

```diff
@@ -20,21 +20,30 @@ func TestProcess_Add(t *testing.T) {
 	proc, err := New(ctx, cancel, logger, WithUniqueID("<id>"))
 	assert.NoError(t, err)
 
-	// Do some work, operations run in parallel, sleep determines the completion order to make it testable
+	op1 := &sync.WaitGroup{}
+	op1.Add(1)
+	op2 := &sync.WaitGroup{}
+	op2.Add(1)
+	op3 := &sync.WaitGroup{}
+	op3.Add(1)
```

Comment:
Nechcel som tieto testy komplikovat, tak som pouzil najprv `sleep`, no `sleep(100ms)` hlavne na Windows nemusi byt dostatocna doba, aby sa dokoncila dalsia goroutine, tak som to radsej upravil. 

(ono cele tieto testy zbehnu za `9ms`, len na Windows to uz raz spadlo na poradi, operacie bezia paralelne, len to musime spravit nejak testovatelne).